src/index.js

```diff
@@ -91,14 +91,20 @@ const renderConditional = tree => {
   }, []);
 };
 
-const indent = (tree, {rules: {indent, eol, blankLines}}) => {
+const indent = (tree, {rules: {indent, eol, blankLines, maxlen}}) => {
   const setIndent = (tree, level = 0) => tree.reduce((previousValue, node, index) => {
     if (typeof node === 'object' && Object.prototype.hasOwnProperty.call(node, 'content')) {
       node.content = setIndent(node.content, ++level);
       --level;
     }
 
     if (tree.length === 1 && typeof tree[index] === 'string') {
+      if (tree[index].length >= maxlen) {
+        let l = level;
+        let e = l;
+        return [...previousValue, getIndent(l, {indent, eol}), node, getIndent(--e, {indent, eol})];
```

Comment:
I think here you can do without variables
```js
        return [
            ...previousValue, 
            getIndent(level, {indent, eol}), 
            node, 
            getIndent(--level, {indent, eol})
        ];
```

Reply:
actually I didnt get the `setIndent` function completely and thought using the `level` might change the indentation to other blocks. that's why I used the copy. I will change it to `level` and check the tests again 👍 

thanks .

Reply:
done 👍